### DISCUSSION TITLE
How do you plan to develop a function that supports text and background colors?

### DISCUSSION THREAD

**Discussion by netager123** [umaranis/svelte-lexical#92] [Q&A] [answered]
Hello sir, @umaranis

I really need color support ASAP on svelte-lexical.
Please let me know that upgrade plan for support to color if you can.

Thank you.

**Comment by umaranis** [comment, marked as answer]
Hi @netager123 ,

I have implemented the color picker for the background and text color with commit fc2a631da3814f97c9603f246f75e1b8d7555186.

I will include it in a new release soon.

**Reply by netager123** [reply]
Thank you very much for your prompt support.
I'm really happy.
I wish you happiness and good health.
